src/components/Trade/TradeRoutes.tsx

```diff
@@ -12,15 +17,60 @@ export const entries = ['/send/details', '/send/confirm']
 export const TradeRoutes = () => {
   const location = useLocation()
   const history = useHistory()
-  const { setValue } = useFormContext()
-  const handleSellClick = (asset: Asset) => {
-    setValue('sellAsset.currency', asset)
-    history.push('/trade/input')
-  }
-  const handleBuyClick = (asset: Asset) => {
-    setValue('buyAsset.currency', asset)
-    history.push('/trade/input')
-  }
+  const { getValues, setValue } = useFormContext<TradeState>()
+  const { getCryptoQuote, getBestSwapper, getDefaultPair } = useSwapper()
+  const buyAsset = getValues('buyAsset')
+  const sellAsset = getValues('sellAsset')
+  const assetService = useAssets()
+
+  const setDefaultAssets = useCallback(async () => {
```

Comment:
Do you think that setDefaultAssets, handleSellClick and handleBuyClick need to be tested?